observation_portal/accounts/urls.py

```diff
@@ -1,11 +1,15 @@
-from django.urls import re_path, include
+from django.urls import re_path, include, path
 from django.views.generic.base import TemplateView
 from registration.backends.default.views import RegistrationView
 
 from observation_portal.accounts.forms import CustomRegistrationForm
 
+from .views import CustomLoginView, CustomPasswordChangeView
+
 urlpatterns = [
     re_path(r'^register/$', RegistrationView.as_view(form_class=CustomRegistrationForm), name='registration_register'),
     re_path(r'^loggedinstate/$', TemplateView.as_view(template_name='auth/logged_in_state.html'), name='logged-in-state'),
+    path("login/", CustomLoginView.as_view(), name="auth_login"),
```

Comment:
Does this just override the /accounts/login/ endpoint that was already there? I assume that one must have been provided by bottom re_path url pattern?

Reply:
Yes, those are provided by `include('registration.backends.default.urls')`. Rather than figure all those out and type them again, I'm just letting these two take precedence by placing them above that include.